yevhenii.dolmatov/task_02/tamagotchi.rb

```diff
@@ -0,0 +1,203 @@
+class Tamagotchi

```

Comment:
@gunnner hey, could you please pull the latest changes from upstream to have rubocop fixes pulled

Reply:
Done.